src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx

```diff
@@ -110,6 +122,55 @@ it("displays the artwork title correctly without date", () => {
   expect(serifChildren(component)).not.toContain(`${saleArtwork.artwork.title},`)
 })
 
+it("can load and display price summary", () => {
+  const component = mountConfirmBidComponent(initialProps)
+
+  expect(component.root.findAllByType(Spinner).length).toEqual(1)
+  ;(defaultEnvironment as any).mock.resolveMostRecentOperation(() => ({
+    data: {
+      node: {
+        __typename: "SaleArtwork",
+        calculatedCost: {
+          buyersPremium: {
+            display: "$9,000.00",
+          },
+          subtotal: {
+            display: "$54,000.00",
+          },
+        },
+      },
+    },
+  }))
+
+  expect(component.root.findAllByType(Spinner).length).toEqual(0)
```

Comment:
If the loader code was working, would this test failure? I guess what is this assertion supposed to test?

Reply:
The `<Spinner>` component is still mountable and this test passes. The only thing that's not working is animation which is encapsulated within this component.